docs/community.md

```diff
@@ -0,0 +1,45 @@
+# Community Resources
+
+!!! Note
+
+    Google and the ADK team do not provide support for the content linked in
+    these external community resources.
+
+Welcome! This page highlights resources maintained by the Agent Development Kit
+community.
+
+## Translations
+
+Community-provided translations of the official documentation.
+
+*   **[adk.wiki - Chinese Documentation](https://adk.wiki/)**
+    > adk.wiki is the Chinese version of the Agent Development Kit
+    > documentation, maintained by an individual. The documentation is
+    > continuously updated and translated to provide a localized reading
+    > experience for developers in China.
+
+## Tutorials, Guides & Blog Posts
+
+*(Find community-written guides covering ADK features, use cases, and
+integrations here.)*
+
+## Videos & Screencasts
+
+*(Discover video walkthroughs, talks, and demos showcasing ADK.)*
+
+## Contributing Your Resource
+
+Have an ADK resource to share (tutorial, translation, tool, video, example)?
+Help grow the ecosystem!
+
+- Submit a Pull Request (PR) to this page in the
+  [`google/adk-docs`](https://github.com/google/adk-docs/pulls) repository.
+- Open an issue in the
+  [`google/adk-docs`](https://github.com/google/adk-docs/issues) repository.
+- In your PR or issue, provide a link, brief description, and category.
+- Refer to the [Contributing Guide](contributing-guide.md) for details
+  (including the CLA).
+  - Want to discuss first? Join our
+    [GitHub Discussions](https://github.com/google/adk-python/discussions).
+
+Thank you for your contributions to Agent Development Kit! ❤️
```

Comment:
This is part of the contributtion-guide. Did you want to move it here?

Reply:
Ah great! In that case I will add a link to the contribution guide at the bottom of the new page and avoid repeating this info. Thanks!